### ISSUE TITLE
Not able to signin to Azure from VSCode to be able to deploy

### ISSUE THREAD

**Issue by RitwikChatterjee** [Microsoft/vscode-azurefunctions#896]
Hi,

I am using v0.13.1 of the extension on a Windows machine. 
I am able to create and test locally. However, in the step where I need to signin to Azure to deploy, I am not able to do so. 

When I click on "Sign in to Azure" in the extension, it takes me to the browser with a code. The browser successfully signs me in and shows me the following message:
"_You have signed in to the Visual Studio Code application on your device. You may now close this window._"

However, when I return to VSCode, it is not signed in. 
I believe the handshake is not happening properly.

**Comment by EricJizbaMSFT** [comment]
What version of the azure account extension do you have?
And what version of VS Code?

**Comment by RitwikChatterjee** [comment]
Azure Account 0.6.2
VS Code 1.29.1

**Comment by EricJizbaMSFT** [comment]
@RitwikChatterjee can you try version 0.8.0 of the account extension? It has a completely re-done experience for signing in. It does require you to be on at least version 1.30.0 of VS Code. Is there any particular reason you're still on 1.29.1 or is that unintentional?

**Comment by RitwikChatterjee** [comment]
Unintentional. Let me try to upgrade

**Comment by RitwikChatterjee** [comment]
I now have VS Code 1.30.2 and Azure Account 0.8.0 and Azure Functions 0.13.1. Same issue.
From a UX perspective, I did not notice any difference with Azure Account 0.8.0 vs 0.6.2. Not sure how it is working internally.

**Comment by EricJizbaMSFT** [comment]
Did you reload VS Code after updating the account extension?

It should automatically open the browser without the notification and you shouldn't have to copy or paste any code.

**Comment by RitwikChatterjee** [comment]
I am not sure what you mean by "reload VS Code". I have restarted it a couple of times. I did have to reload Azure Account extension. However, from the UX perspective I am still seeing this:

![image](https://user-images.githubusercontent.com/5916320/50988212-2730c380-14c9-11e9-8fcf-a722e37fcce6.png)

Here is a view of my enabled extensions:
![image](https://user-images.githubusercontent.com/5916320/50988280-5f380680-14c9-11e9-8832-c08ac702c101.png)

**Comment by RitwikChatterjee** [comment]
Visual Studio details:

Version: 1.30.2 (system setup)
Commit: 61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8
Date: 2019-01-07T22:54:13.295Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
OS: Windows_NT x64 10.0.15063

**Comment by RitwikChatterjee** [comment]
Hi Eric, 
Thanks for your support so far. I did try uninstalling the extensions from VS Code and remove them from Extension root. This is what I did:

Uninstalled Azure Functions extension from VS Code. Then delete the corresponding `ms-vscode.azure*` folders from the extension root. Here is what I am starting with right now:

![image](https://user-images.githubusercontent.com/5916320/50989885-cce63180-14cd-11e9-888e-c6dd0e6e2d3f.png)

I reinstalled Azure Functions 0.13.1 extension in VS Code. I restarted VS Code. The installation of Azure Functions 0.13.1 automatically installs Azure Accounts 0.8.0. So here is where I land:
![image](https://user-images.githubusercontent.com/5916320/50990172-9230c900-14ce-11e9-91ed-a5aad0d74ab5.png)

Here is how the Extensions root look like now:
![image](https://user-images.githubusercontent.com/5916320/50990211-ac6aa700-14ce-11e9-9890-3502ca370601.png)

Now I click on Azure Functions extension and click on _"Sign in to Azure..."_
I immediately see the following notification on the bottom right:

![image](https://user-images.githubusercontent.com/5916320/50990325-f8b5e700-14ce-11e9-9ec8-241cea885094.png)

So, for some reason, I am not seeing a change in the UX, and the result is the same - not able to log in.
